src/stores/leveldown.ts

```diff
@@ -23,9 +23,14 @@ class LeveldownStore implements StoreInstance {
     this.db = levelup(leveldown(path))
   }
 
-  async get (key: string) {
+  async get (key: string): Promise<string | undefined> {
```

Comment:
Why does this have `| undefined`?

Reply:
if the key doesn't exist then it returns `undefined`